pkg/opentelemetry/opentelemetrytraces/storage_test.go

```diff
@@ -72,7 +72,7 @@ func TestStore(t *testing.T) {
 }
 
 func testAppendLog(t *testing.T, store storage.Store) {
-	err := store.AppendLog(context.Background(), core.NewTransactionLog(nil, core.Transaction{}))
+	err := store.Commit(context.Background(), core.Transaction{})
```

Comment:
This test could be deleted, doesn't make sense anymore

Reply:
Yes